docs/platforms/javascript/common/best-practices/shared-environments.mdx

```diff
@@ -130,4 +130,4 @@ You'd risk quota increases with unactionable issues, interference with other Sen
 </Alert>
 
 This option is purely meant as an escape hatch if our browser extension check is incorrectly detecting a browser extension.
-An example for this might be a
+An example for this might be a cross-platform application framework that exposes similar global APIs like browser a browser extension environment.
```

Comment:
Something is off here or am I missing something? 🤔 
"like browser a browser extension"